### ISSUE TITLE
Font related property broken

### ISSUE THREAD

**Issue by mai1015** [tamagui/tamagui#2603]
### Current Behavior

after upgrading with `yarn upgrade:tamagui`
When run `yarn web:prod` the nodejs compile failed with following log

```
Type error: Type '"$8"' is not assignable to type '"unset" | GetThemeValueForKey<"lineHeight"> | null | undefined'.

  64 |         scale={1}
  65 |         size="$6"
> 66 |         lh="$8"
     |         ^
  67 |         ta="center"
  68 |         col="$color9"
  69 |         selectable={false}
   Linting and checking validity of types  .
   ```

### Expected Behavior

It should compile correctly

### Tamagui Version

```markdown
1.95.3
```

### Platform (Web, iOS, Android)

```markdown
Web
```

### Reproduction

```markdown
start a new project in takeout
yarn upgrade:tamagui
yarn web:prod
```

### System Info

```markdown
System:
    OS: Windows 11 10.0.22631
    CPU: (24) x64 12th Gen Intel(R) Core(TM) i9-12900HX
    Memory: 28.45 GB / 63.74 GB
  Binaries:
    Node: 18.17.0 - C:\Program Files\nodejs\node.EXE
    Yarn: 4.1.0 - ~\AppData\Roaming\npm\yarn.CMD
    npm: 9.6.7 - C:\Program Files\nodejs\npm.CMD
    pnpm: 8.6.12 - ~\AppData\Local\pnpm\pnpm.EXE
  Browsers:
    Edge: Chromium (123.0.2420.97)
    Internet Explorer: 11.0.22621.1
  npmPackages:
    @babel/runtime: ^7.18.9 => 7.23.9
    @manypkg/cli: ^0.19.1 => 0.19.2
    @nderscore/tamagui-typescript-plugin: ^0.6.0 => 0.6.0
    @tamagui/cli: ^1.95.3 => 1.95.3
    @turbo/gen: ^1.10.12 => 1.10.13
    change-case: ^4.1.2 => 4.1.2
    check-dependency-version-consistency: ^3.0.3 => 3.3.0
    dotenv: ^16.0.3 => 16.4.5
    dotenv-cli: ^6.0.0 => 6.0.0
    eslint: ^8.46.0 => 8.49.0
    node-gyp: ^9.3.1 => 9.4.0
    prettier: ^2.7.1 => 2.8.8
    turbo: ^1.10.14 => 1.10.14
    typescript: ^5.3.3 => 5.3.3
```

**Comment by TheEhsanSarshar** [comment]
it seems that lineHeight `8` is missing in your tamagui config

**Comment by mai1015** [comment]
> it seems that lineHeight `8` is missing in your tamagui config make sure tamagui configs are correct, I wasn't able to reproduce it tho

I did not change anything in the takeout template. before updating the tamagui, everything works fine, but after upgrading it. the compile failed

**Comment by mai1015** [comment]
looks like it is fixed in 1.96.0
